internal/provider/launchpad_config_resource.go

```diff
@@ -18,19 +17,19 @@ type LaunchpadConfigResource struct {
 	testingMode bool
 }
 
-func NewLaunchpadConfigResource() resource.Resource {
+func NewLaunchpadConfigResource() resource.Resource { //nolint:ireturn
```

Comment:
The function returns an interface so that it matches the interface for the resource